docs/audit/IOS_DOCS_LEDGER_ALIGNMENT_AUDIT_2026-02-06.md

```
@@ -0,0 +1,191 @@
+```
+
+  - **Output (raw)**:
+
+```text
+38:  // TODO: Backend endpoint /api/nutrition/{date} not yet implemented (GitHub issue)
+57:      let path = "api/nutrition/\\(dateString)"
+```
+
+  - **Expected snippet:** iOS client currently builds path `api/nutrition/<date>` and contains a TODO claiming it is not implemented (docs must forbid this as SoT).
+
+- **Repo truth (backend)**
```

markdownlint-cli2 (0.20.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 0

(MD005, list-indent)